Tests/SentryTests/Integrations/SentrySessionTrackerTests.swift

```diff
@@ -28,15 +28,22 @@ class SentrySessionTrackerTests: XCTestCase {
     }
     
     private var fixture: Fixture!
+    private var sut: SessionTracker!
 
     override func setUp() {
         super.setUp()
 
         fixture = Fixture()
+        sut = fixture.getSut()
```

Comment:
L: the downside of doing this here is, if you need 2 instances of a `sut` for a specific test case, you'd need to reuse an already created var, I think it's fine for this class, I don't see a case where we'd need this anyway, but keep in mind about other testing classes.

Reply:
Yes, this is the downside and we can change when needed. Otherwise I would have `sut = fixture.getSut()` and `sut.stop()` in every test method.